README.md

```diff
@@ -136,15 +136,20 @@ Detailed doc is [here](https://firebase.google.com/docs/cli/auth).
 
 ### Cloud Functions Commands
 
-| Command                    | Description                                                                                                  |
-| -------------------------- | ------------------------------------------------------------------------------------------------------------ |
-| **functions:log**          | Read logs from deployed Cloud Functions.                                                                     |
-| **functions:config:set**   | Store runtime configuration values for the current project's Cloud Functions.                                |
-| **functions:config:get**   | Retrieve existing configuration values for the current project's Cloud Functions.                            |
-| **functions:config:unset** | Remove values from the current project's runtime configuration.                                              |
-| **functions:config:clone** | Copy runtime configuration from one project environment to another.                                          |
-| **functions:delete**       | Delete one or more Cloud Functions by name or group name.                                                    |
-| **functions:shell**        | Locally emulate functions and start Node.js shell where these local functions can be invoked with test data. |
+| Command                       | Description                                                                                                  |
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------ |
+| **functions:log**             | Read logs from deployed Cloud Functions.                                                                     |
```

Comment:
If you wanted to be exhaustive, we also have this command you could add here:
```
| **functions:list** | List all deployed functions in your Firebase project. |
```

Reply:
Thank you for your feedback.
I add functions:list in list.